qdrant-landing/content/documentation/concepts/points.md

```diff
@@ -1632,12 +1631,97 @@ All resulting points are sorted by ID. To query the next page it is necessary to
 For convenience, this ID is also returned in the field `next_page_offset`.
 If the value of the `next_page_offset` field is `null` - the last page is reached.
 
-<!-- 
-Python client:
+### Order points by a payload key
+
+_Available as of v1.8_
+
+When using the [`scroll`](#scroll-points) API, it is possible to sort the results by the contents of a payload field. This is useful when you want to retrieve points in a specific order, for example, by a timestamp, for chronological order.
+
+<aside role="status">Without an appropriate index, custom ordering would be a very expensive operation. For this reason, in Qdrant, it is a requirement to have an index capable of <a href=/documentation/concepts/indexing/#payload-index target="_blank">Range filtering conditions</a> on the field you want to `order_by`. Once the index is set up, these ordering operations can be really cheap.</aside>
```

Comment:
By "expensive," I'm guessing you mean as in load on the system (not actual cost). I therefore suggest:

```suggestion
<aside role="status">Without an appropriate index, custom ordering can create costly loads. Qdrant therefore requires an index which supports <a href=/documentation/concepts/indexing/#payload-index target="_blank">Range filtering conditions</a> on one ore more specific `order_by` fields.</aside>
```

Reply:
"costly" is not the same as "expensive"?

Reply:
> "costly" is not the same as "expensive"?

I asked ChatGPT the question, and I like the answer:

![Screenshot from 2024-02-22 12-25-53](https://github.com/qdrant/landing_page/assets/3287976/7eb945cd-1840-4c3c-969b-254201a56543)